conda/meta.yaml

```diff
@@ -75,7 +75,7 @@ requirements:
     - metis ==5.1.0
     - mpi4py
     - pynastran
-    - numba
+    - numba >=0.55.2
```

Comment:
Should we add the same minimum version to the setup.py requirements?

Reply:
I dont think so. This appeared to be an issue specifically with the conda packages dependencies for older versions of numba that didn't have a version upperbound for newer numpy versions. This could lead to a situation where an old version of numba could be installed with a new (incompatible) version of numpy.

Reply:
ok, if it's a very conda-specific issue then I guess we can go with this as is